File: Sample/java/src/main/java/net/nend/sample/java/banner/adjust/AdjustSizeActivity.java
package net.nend.sample.java.banner.adjust;

import android.os.Bundle;

import net.nend.sample.java.R;
import net.nend.sample.java.SimpleListActivity;

import java.util.ArrayList;
import java.util.List;

public class AdjustSizeActivity extends SimpleListActivity {

	private static final List<String> SAMPLE_ACTIVITIES = new ArrayList<String>() {
		{
			add(AdjustSizeXmlLayoutActivity.class.getName());
			add(AdjustSizeJavaCallActivity.class.getName());
		}
	};

	@Override
	public void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		instantiateMenuListFragment(R.layout.adjust_size_sample, SAMPLE_ACTIVITIES);
	}
}

File: Sample/kotlin/src/main/java/net/nend/sample/kotlin/banner/adjust/AdjustSizeActivity.kt
package net.nend.sample.kotlin.banner.adjust

import android.os.Bundle
import net.nend.sample.kotlin.R
import net.nend.sample.kotlin.SimpleListActivity
import java.util.*

class AdjustSizeActivity : SimpleListActivity() {

    private val list: ArrayList<String> = object : ArrayList<String>() {
        init {
            add(AdjustSizeXmlLayoutActivity::class.java.name)
            add(AdjustSizeNoXmlActivity::class.java.name)
        }
    }

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        instantiateMenuListFragment(R.layout.adjust_size_sample, list)
    }
}